README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
 * Command line development tools (`make`, `git`, and a compiler) for your platform
```

Comment:
Hmm. As far as I see, you added that comment relatively recently in 3f5d7ec7. It's not really important, though. I'm just curious whether or not I'm missing something. (Maybe some dependency needs to be built from source on some OS'es? phantomjs?)